service/src/main/java/eu/europa/ec/fisheries/uvms/activity/fa/dao/proxy/FaCatchSummaryCustomChildProxy.java

```diff
@@ -17,107 +17,6 @@ This file is part of the Integrated Fisheries Data Management (IFDM) Suite. The
  */
 public class FaCatchSummaryCustomChildProxy extends FaCatchSummaryCustomProxy {
 
-    // Override these methods as reflection code will fail if not explicitely specified
```

Comment:
Feels like maybe we can just throw the whole class away?

Reply:
That's a bit more than a sonar fix